content/docs/learn/typed-errors/from-either-to-raise.md

```diff
@@ -0,0 +1,181 @@
+---
+title: From Either to Raise
+sidebar_position: 5
+description: Learning about Raise from other FP ecosystems.
+---
+
+# From `Either` to `Raise`
+
+Typed errors in other functional ecosystems usually revolve around
+special types like `Either`, which provide the ability to describe
```

Comment:
```suggestion
abstract types like `Either`, which provide the ability to describe
```

Or maybe general types?  I think the point here is that Either is a very generic tool that gets used for error handling, as opposed to a tool for error handling.